go/kbfs/libkbfs/folder_block_manager_test.go

```diff
@@ -176,6 +184,43 @@ func TestQuotaReclamationUnembedded(t *testing.T) {
 	}
 }
 
+// Just like the simple case, except tests that it unembeds large sets
+// of pointers correctly.
+func TestQuotaReclamationUnembeddedJournal(t *testing.T) {
+	var userName kbname.NormalizedUsername = "test_user"
+	config, _, ctx, cancel := kbfsOpsInitNoMocks(t, userName)
+	defer kbfsTestShutdownNoMocks(ctx, t, config, cancel)
+
+	tempdir, err := ioutil.TempDir(os.TempDir(), "journal_server")
+	require.NoError(t, err)
+	defer ioutil.RemoveAll(tempdir)
+
+	config.EnableDiskLimiter(tempdir)
+	require.NoError(t, err)
+	err = config.EnableJournaling(
+		ctx, tempdir, TLFJournalBackgroundWorkPaused)
+	require.NoError(t, err)
+
+	config.bsplit.(*data.BlockSplitterSimple).
+		SetBlockChangeEmbedMaxSizeForTesting(32)
+
+	rootNode := GetRootNodeOrBust(
+		ctx, t, config, userName.String(), tlf.Private)
+	jManager, err := GetJournalManager(config)
+	require.NoError(t, err)
+	jManager.PauseBackgroundWork(ctx, rootNode.GetFolderBranch().Tlf)
+
+	ops, _ := testQuotaReclamation(ctx, t, config, userName)
+
+	t.Log("Check that the latest merged revision didn't get updated")
```

Comment:
So we're just using the merged revision as a proxy check for unembedding?

Reply:
No, that's actually the whole point of this check.  We know they'll be unembedded because of the `SetBlockChangeEmbedMaxSizeForTesting()` call above.  But the bad merged revision was one of the things causing lots of problems for the user, so that's why I'm checking it directly here.

Reply:
Got it, thanks.